packages/admin-ui-core/src/createTheme.ts

```diff
@@ -115,6 +115,79 @@ function createRootStylesAsObject(cssVariables: CSSVariables) {
   }, {})
 }
 
+/**
+ * Return the custom theme which will override the initial theme.
+ * The custom theme will be loaded from the admin-ui.config.js file found in the project.
+ */
+
+export function getCustomTheme(
+  configPath = 'admin-ui.config.js'
+): Record<string, any> {
+  let customTheme = {}
+
+  try {
+    // eslint-disable-next-line node/global-require, @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
+    const { theme } = require(`${process.cwd()}/${configPath}`)
+
+    customTheme = theme
+  } catch (err) {
+    console.log('There is no theme config file')
+  }
+
+  return customTheme
+}
+
+/**
+ * Merge deeply two theme objects, overriding the initial theme props by the other one props.
+ * @example
+ * mergeThemes
+ * {
+ *  colors: {
+ *    background: 'blue',
+ *    fg: 'black',
+ *    text: 'black',
+ *  },
+ * },
+ * {
+ *  colors: {
+ *    background: 'red',
+ *    fg: 'red'
+ *  }
+ * })
+ *
+ * // returns:
+ * {
+ *  colors: {
+ *    background: 'red',
+ *    fg: 'red',
+ *    text: 'black',
+ *  }
+ * }
+ */
+
+export function mergeThemes(
```

Comment:
You could just use `merge` from `@vtex/admin-ui-util`.

Reply:
ok, I will check it!!

Reply:
It worked, but I had to consider when `global: {}`, a way of disabling global styles